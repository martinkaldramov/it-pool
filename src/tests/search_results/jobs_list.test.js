import React from 'react';
import { shallow } from 'enzyme';
import JobsList from '../../components/search_results/jobs_list';

describe('JobsList', () => {
  const wrap = shallow(<JobsList />);

  it('should render successfully', () => {
    expect(wrap).toMatchSnapshot();
  });

  it('should have 3 children', () => {
    expect(wrap.children).toBe(3);
  });
});
