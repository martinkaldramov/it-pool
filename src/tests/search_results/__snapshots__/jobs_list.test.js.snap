// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobsList should render successfully 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <JobsList />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <ul>
        <Job
                jobData={
                        Object {
                                "category": "Web Development",
                                "city": "Sofia",
                                "company": "MODIS BULGARIA",
                                "date": 18,
                                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                "experience": "Solid",
                                "skills": Array [
                                  "React.JS",
                                  "Angular.io",
                                  "DevOps",
                                  "ExpressJS",
                                  "Google Firebase",
                                ],
                                "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                "type": "Full Time",
                                "views": 236,
                              }
                }
        />
        <Job
                jobData={
                        Object {
                                "category": "Desktop Programming",
                                "city": "Plovdiv",
                                "company": "Software Group",
                                "date": 18,
                                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                "experience": "Medium",
                                "skills": Array [
                                  "C#",
                                  ".NET",
                                  "Visual Studio",
                                  "ExpressJS",
                                  "Google Firebase",
                                ],
                                "title": "Ericsson FixIT Developer",
                                "type": "Part Time",
                                "views": 1006,
                              }
                }
        />
        <Job
                jobData={
                        Object {
                                "category": "Web Development",
                                "city": "Sofia",
                                "company": "MODIS BULGARIA",
                                "date": 18,
                                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                "experience": "Solid",
                                "skills": Array [
                                  "React.JS",
                                  "Angular.io",
                                  "DevOps",
                                  "ExpressJS",
                                  "Google Firebase",
                                ],
                                "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                "type": "Full Time",
                                "views": 236,
                              }
                }
        />
        <Job
                jobData={
                        Object {
                                "category": "Web Development",
                                "city": "Sofia",
                                "company": "MODIS BULGARIA",
                                "date": 18,
                                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                "experience": "Solid",
                                "skills": Array [
                                  "React.JS",
                                  "Angular.io",
                                  "DevOps",
                                  "ExpressJS",
                                  "Google Firebase",
                                ],
                                "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                "type": "Full Time",
                                "views": 236,
                              }
                }
        />
</ul>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Job
            jobData={
                        Object {
                                    "category": "Web Development",
                                    "city": "Sofia",
                                    "company": "MODIS BULGARIA",
                                    "date": 18,
                                    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                    "experience": "Solid",
                                    "skills": Array [
                                      "React.JS",
                                      "Angular.io",
                                      "DevOps",
                                      "ExpressJS",
                                      "Google Firebase",
                                    ],
                                    "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                    "type": "Full Time",
                                    "views": 236,
                                  }
            }
/>,
          <Job
            jobData={
                        Object {
                                    "category": "Desktop Programming",
                                    "city": "Plovdiv",
                                    "company": "Software Group",
                                    "date": 18,
                                    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                    "experience": "Medium",
                                    "skills": Array [
                                      "C#",
                                      ".NET",
                                      "Visual Studio",
                                      "ExpressJS",
                                      "Google Firebase",
                                    ],
                                    "title": "Ericsson FixIT Developer",
                                    "type": "Part Time",
                                    "views": 1006,
                                  }
            }
/>,
          <Job
            jobData={
                        Object {
                                    "category": "Web Development",
                                    "city": "Sofia",
                                    "company": "MODIS BULGARIA",
                                    "date": 18,
                                    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                    "experience": "Solid",
                                    "skills": Array [
                                      "React.JS",
                                      "Angular.io",
                                      "DevOps",
                                      "ExpressJS",
                                      "Google Firebase",
                                    ],
                                    "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                    "type": "Full Time",
                                    "views": 236,
                                  }
            }
/>,
          <Job
            jobData={
                        Object {
                                    "category": "Web Development",
                                    "city": "Sofia",
                                    "company": "MODIS BULGARIA",
                                    "date": 18,
                                    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                    "experience": "Solid",
                                    "skills": Array [
                                      "React.JS",
                                      "Angular.io",
                                      "DevOps",
                                      "ExpressJS",
                                      "Google Firebase",
                                    ],
                                    "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                    "type": "Full Time",
                                    "views": 236,
                                  }
            }
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "jobData": Object {
              "category": "Web Development",
              "city": "Sofia",
              "company": "MODIS BULGARIA",
              "date": 18,
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
              "experience": "Solid",
              "skills": Array [
                "React.JS",
                "Angular.io",
                "DevOps",
                "ExpressJS",
                "Google Firebase",
              ],
              "title": "Proofreading/light editing of a legal document translated from Japanese to English",
              "type": "Full Time",
              "views": 236,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "jobData": Object {
              "category": "Desktop Programming",
              "city": "Plovdiv",
              "company": "Software Group",
              "date": 18,
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
              "experience": "Medium",
              "skills": Array [
                "C#",
                ".NET",
                "Visual Studio",
                "ExpressJS",
                "Google Firebase",
              ],
              "title": "Ericsson FixIT Developer",
              "type": "Part Time",
              "views": 1006,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "jobData": Object {
              "category": "Web Development",
              "city": "Sofia",
              "company": "MODIS BULGARIA",
              "date": 18,
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
              "experience": "Solid",
              "skills": Array [
                "React.JS",
                "Angular.io",
                "DevOps",
                "ExpressJS",
                "Google Firebase",
              ],
              "title": "Proofreading/light editing of a legal document translated from Japanese to English",
              "type": "Full Time",
              "views": 236,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "jobData": Object {
              "category": "Web Development",
              "city": "Sofia",
              "company": "MODIS BULGARIA",
              "date": 18,
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
              "experience": "Solid",
              "skills": Array [
                "React.JS",
                "Angular.io",
                "DevOps",
                "ExpressJS",
                "Google Firebase",
              ],
              "title": "Proofreading/light editing of a legal document translated from Japanese to English",
              "type": "Full Time",
              "views": 236,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "ul",
    },
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <ul>
          <Job
                    jobData={
                              Object {
                                        "category": "Web Development",
                                        "city": "Sofia",
                                        "company": "MODIS BULGARIA",
                                        "date": 18,
                                        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                        "experience": "Solid",
                                        "skills": Array [
                                          "React.JS",
                                          "Angular.io",
                                          "DevOps",
                                          "ExpressJS",
                                          "Google Firebase",
                                        ],
                                        "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                        "type": "Full Time",
                                        "views": 236,
                                      }
                    }
          />
          <Job
                    jobData={
                              Object {
                                        "category": "Desktop Programming",
                                        "city": "Plovdiv",
                                        "company": "Software Group",
                                        "date": 18,
                                        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                        "experience": "Medium",
                                        "skills": Array [
                                          "C#",
                                          ".NET",
                                          "Visual Studio",
                                          "ExpressJS",
                                          "Google Firebase",
                                        ],
                                        "title": "Ericsson FixIT Developer",
                                        "type": "Part Time",
                                        "views": 1006,
                                      }
                    }
          />
          <Job
                    jobData={
                              Object {
                                        "category": "Web Development",
                                        "city": "Sofia",
                                        "company": "MODIS BULGARIA",
                                        "date": 18,
                                        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                        "experience": "Solid",
                                        "skills": Array [
                                          "React.JS",
                                          "Angular.io",
                                          "DevOps",
                                          "ExpressJS",
                                          "Google Firebase",
                                        ],
                                        "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                        "type": "Full Time",
                                        "views": 236,
                                      }
                    }
          />
          <Job
                    jobData={
                              Object {
                                        "category": "Web Development",
                                        "city": "Sofia",
                                        "company": "MODIS BULGARIA",
                                        "date": 18,
                                        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                        "experience": "Solid",
                                        "skills": Array [
                                          "React.JS",
                                          "Angular.io",
                                          "DevOps",
                                          "ExpressJS",
                                          "Google Firebase",
                                        ],
                                        "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                        "type": "Full Time",
                                        "views": 236,
                                      }
                    }
          />
</ul>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Job
              jobData={
                            Object {
                                          "category": "Web Development",
                                          "city": "Sofia",
                                          "company": "MODIS BULGARIA",
                                          "date": 18,
                                          "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                          "experience": "Solid",
                                          "skills": Array [
                                            "React.JS",
                                            "Angular.io",
                                            "DevOps",
                                            "ExpressJS",
                                            "Google Firebase",
                                          ],
                                          "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                          "type": "Full Time",
                                          "views": 236,
                                        }
              }
/>,
            <Job
              jobData={
                            Object {
                                          "category": "Desktop Programming",
                                          "city": "Plovdiv",
                                          "company": "Software Group",
                                          "date": 18,
                                          "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                          "experience": "Medium",
                                          "skills": Array [
                                            "C#",
                                            ".NET",
                                            "Visual Studio",
                                            "ExpressJS",
                                            "Google Firebase",
                                          ],
                                          "title": "Ericsson FixIT Developer",
                                          "type": "Part Time",
                                          "views": 1006,
                                        }
              }
/>,
            <Job
              jobData={
                            Object {
                                          "category": "Web Development",
                                          "city": "Sofia",
                                          "company": "MODIS BULGARIA",
                                          "date": 18,
                                          "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                          "experience": "Solid",
                                          "skills": Array [
                                            "React.JS",
                                            "Angular.io",
                                            "DevOps",
                                            "ExpressJS",
                                            "Google Firebase",
                                          ],
                                          "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                          "type": "Full Time",
                                          "views": 236,
                                        }
              }
/>,
            <Job
              jobData={
                            Object {
                                          "category": "Web Development",
                                          "city": "Sofia",
                                          "company": "MODIS BULGARIA",
                                          "date": 18,
                                          "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                                          "experience": "Solid",
                                          "skills": Array [
                                            "React.JS",
                                            "Angular.io",
                                            "DevOps",
                                            "ExpressJS",
                                            "Google Firebase",
                                          ],
                                          "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                                          "type": "Full Time",
                                          "views": 236,
                                        }
              }
/>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "jobData": Object {
                "category": "Web Development",
                "city": "Sofia",
                "company": "MODIS BULGARIA",
                "date": 18,
                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                "experience": "Solid",
                "skills": Array [
                  "React.JS",
                  "Angular.io",
                  "DevOps",
                  "ExpressJS",
                  "Google Firebase",
                ],
                "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                "type": "Full Time",
                "views": 236,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "jobData": Object {
                "category": "Desktop Programming",
                "city": "Plovdiv",
                "company": "Software Group",
                "date": 18,
                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                "experience": "Medium",
                "skills": Array [
                  "C#",
                  ".NET",
                  "Visual Studio",
                  "ExpressJS",
                  "Google Firebase",
                ],
                "title": "Ericsson FixIT Developer",
                "type": "Part Time",
                "views": 1006,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "jobData": Object {
                "category": "Web Development",
                "city": "Sofia",
                "company": "MODIS BULGARIA",
                "date": 18,
                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                "experience": "Solid",
                "skills": Array [
                  "React.JS",
                  "Angular.io",
                  "DevOps",
                  "ExpressJS",
                  "Google Firebase",
                ],
                "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                "type": "Full Time",
                "views": 236,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "jobData": Object {
                "category": "Web Development",
                "city": "Sofia",
                "company": "MODIS BULGARIA",
                "date": 18,
                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent sit amet facilisis ligula, ac consectetur nisl. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce venenatis condimentum tristique. Quisque volutpat vulputate dolor, id rhoncus erat euismod id. Sed maximus dignissim arcu ac pretium. Curabitur vel ante at nunc sodales consectetur. Proin et porta sem, ac volutpat tellus. Curabitur ipsum lacus, gravida id fermentum id, pellentesque vitae erat.",
                "experience": "Solid",
                "skills": Array [
                  "React.JS",
                  "Angular.io",
                  "DevOps",
                  "ExpressJS",
                  "Google Firebase",
                ],
                "title": "Proofreading/light editing of a legal document translated from Japanese to English",
                "type": "Full Time",
                "views": 236,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "ul",
      },
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
